README.md
# bogmat-test

content/history.md
---
title: History
description: Our history page
menu: footer
weight: 20
date: 2018-02-09T19:21:11Z
image:
imageAlt:
draft: false
---


themes/starter/static/assets/app.js
document.addEventListener("DOMContentLoaded",function(){document.body.classList.remove("no-js"),document.body.classList.add("js")}),function(){if(sessionStorage.fontsLoadedFoutWithClassPolyfill)document.documentElement.className+=" fonts-loaded";else{!function(){"use strict";var t,e=[];function n(){for(;e.length;)e[0](),e.shift()}function o(t){this.a=i,this.b=void 0,this.f=[];var e=this;try{t(function(t){!function t(e,n){if(e.a==i){if(n==e)throw new TypeError;var o=!1;try{var s=n&&n.then;if(null!=n&&"object"==typeof n&&"function"==typeof s)return void s.call(n,function(n){o||t(e,n),o=!0},function(t){o||a(e,t),o=!0})}catch(t){return void(o||a(e,t))}e.a=0,e.b=n,r(e)}}(e,t)},function(t){a(e,t)})}catch(t){a(e,t)}}t=function(){setTimeout(n)};var i=2;function s(t){return new o(function(e){e(t)})}function a(t,e){if(t.a==i){if(e==t)throw new TypeError;t.a=1,t.b=e,r(t)}}function r(n){var o;o=function(){if(n.a!=i)for(;n.f.length;){var t=(s=n.f.shift())[0],e=s[1],o=s[2],s=s[3];try{0==n.a?o("function"==typeof t?t.call(void 0,n.b):n.b):1==n.a&&("function"==typeof e?o(e.call(void 0,n.b)):s(n.b))}catch(t){s(t)}}},e.push(o),1==e.length&&t()}o.prototype.g=function(t){return this.c(void 0,t)},o.prototype.c=function(t,e){var n=this;return new o(function(o,i){n.f.push([t,e,o,i]),r(n)})},window.Promise||(window.Promise=o,window.Promise.resolve=s,window.Promise.reject=function(t){return new o(function(e,n){n(t)})},window.Promise.race=function(t){return new o(function(e,n){for(var o=0;o<t.length;o+=1)s(t[o]).c(e,n)})},window.Promise.all=function(t){return new o(function(e,n){function o(n){return function(o){a[n]=o,(i+=1)==t.length&&e(a)}}var i=0,a=[];0==t.length&&e(a);for(var r=0;r<t.length;r+=1)s(t[r]).c(o(r),n)})},window.Promise.prototype.then=o.prototype.c,window.Promise.prototype.catch=o.prototype.g)}(),function(){function t(t,e){document.addEventListener?t.addEventListener("scroll",e,!1):t.attachEvent("scroll",e)}function e(t){this.a=document.createElement("div"),this.a.setAttribute("aria-hidden","true"),this.a.appendChild(document.createTextNode(t)),this.b=document.createElement("span"),this.c=document.createElement("span"),this.h=document.createElement("span"),this.f=document.createElement("span"),this.g=-1,this.b.style.cssText="max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;",this.c.style.cssText="max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;",this.f.style.cssText="max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;",this.h.style.cssText="display:inline-block;width:200%;height:200%;font-size:16px;max-width:none;",this.b.appendChild(this.h),this.c.appendChild(this.f),this.a.appendChild(this.b),this.a.appendChild(this.c)}function n(t,e){t.a.style.cssText="max-width:none;min-width:20px;min-height:20px;display:inline-block;overflow:hidden;position:absolute;width:auto;margin:0;padding:0;top:-999px;white-space:nowrap;font-synthesis:none;font:"+e+";"}function o(t){var e=t.a.offsetWidth,n=e+100;return t.f.style.width=n+"px",t.c.scrollLeft=n,t.b.scrollLeft=t.b.scrollWidth+100,t.g!==e&&(t.g=e,!0)}function i(e,n){function i(){var t=s;o(t)&&t.a.parentNode&&n(t.g)}var s=e;t(e.b,i),t(e.c,i),o(e)}function s(t,e){var n=e||{};this.family=t,this.style=n.style||"normal",this.weight=n.weight||"normal",this.stretch=n.stretch||"normal"}var a=null,r=null,c=null,d=null;function l(){return null===d&&(d=!!document.fonts),d}function f(t,e){return[t.style,t.weight,function(){if(null===c){var t=document.createElement("div");try{t.style.font="condensed 100px sans-serif"}catch(t){}c=""!==t.style.font}return c}()?t.stretch:"","100px",e].join(" ")}s.prototype.load=function(t,o){var s=this,c=t||"BESbswy",d=0,u=o||3e3,h=(new Date).getTime();return new Promise(function(t,o){if(l()&&!function(){if(null===r)if(l()&&/Apple/.test(window.navigator.vendor)){var t=/AppleWebKit\/([0-9]+)(?:\.([0-9]+))(?:\.([0-9]+))/.exec(window.navigator.userAgent);r=!!t&&603>parseInt(t[1],10)}else r=!1;return r}()){var p=new Promise(function(t,e){!function n(){(new Date).getTime()-h>=u?e():document.fonts.load(f(s,'"'+s.family+'"'),c).then(function(e){1<=e.length?t():setTimeout(n,25)},function(){e()})}()}),m=new Promise(function(t,e){d=setTimeout(e,u)});Promise.race([m,p]).then(function(){clearTimeout(d),t(s)},function(){o(s)})}else w=function(){function r(){var e;(e=-1!=w&&-1!=v||-1!=w&&-1!=y||-1!=v&&-1!=y)&&((e=w!=v&&w!=y&&v!=y)||(null===a&&(e=/AppleWebKit\/([0-9]+)(?:\.([0-9]+))/.exec(window.navigator.userAgent),a=!!e&&(536>parseInt(e[1],10)||536===parseInt(e[1],10)&&11>=parseInt(e[2],10))),e=a&&(w==b&&v==b&&y==b||w==g&&v==g&&y==g||w==x&&v==x&&y==x)),e=!e),e&&(E.parentNode&&E.parentNode.removeChild(E),clearTimeout(d),t(s))}var l=new e(c),p=new e(c),m=new e(c),w=-1,v=-1,y=-1,b=-1,g=-1,x=-1,E=document.createElement("div");E.dir="ltr",n(l,f(s,"sans-serif")),n(p,f(s,"serif")),n(m,f(s,"monospace")),E.appendChild(l.a),E.appendChild(p.a),E.appendChild(m.a),document.body.appendChild(E),b=l.a.offsetWidth,g=p.a.offsetWidth,x=m.a.offsetWidth,function t(){if((new Date).getTime()-h>=u)E.parentNode&&E.parentNode.removeChild(E),o(s);else{var e=document.hidden;!0!==e&&void 0!==e||(w=l.a.offsetWidth,v=p.a.offsetWidth,y=m.a.offsetWidth,r()),d=setTimeout(t,50)}}(),i(l,function(t){w=t,r()}),n(l,f(s,'"'+s.family+'",sans-serif')),i(p,function(t){v=t,r()}),n(p,f(s,'"'+s.family+'",serif')),i(m,function(t){y=t,r()}),n(m,f(s,'"'+s.family+'",monospace'))},document.body?w():document.addEventListener?document.addEventListener("DOMContentLoaded",function t(){document.removeEventListener("DOMContentLoaded",t),w()}):document.attachEvent("onreadystatechange",function t(){"interactive"!=document.readyState&&"complete"!=document.readyState||(document.detachEvent("onreadystatechange",t),w())});var w})},"object"==typeof module?module.exports=s:(window.FontFaceObserver=s,window.FontFaceObserver.prototype.load=s.prototype.load)}();var t=new FontFaceObserver("OpenSans"),e=new FontFaceObserver("OpenSansBold",{weight:700});Promise.all([t.load(null,1e4),e.load(null,1e4)]).then(function(){document.documentElement.className+=" fonts-loaded",sessionStorage.fontsLoadedFoutWithClassPolyfill=!0})}}();

themes/starter/src/02-base/js/fontface.js
/* ==========================================================================
   FontFaceObserver loading listener
   ========================================================================== */

/**
 * Adds class of 'fonts-loaded' to body tag when fonts loaded
 * This is taken verbatim from Zach Leatherman @zachleat
 * as includes optimisation for repeat views
 * https://www.zachleat.com/web-fonts/demos/fout-with-class-polyfill.html
 */

(function () {

  // Optimization for Repeat Views
  if (sessionStorage.fontsLoadedFoutWithClassPolyfill) {
    document.documentElement.className += " fonts-loaded";
    return;
  }

  /* Font Face Observer v2.0.13 - © Bram Stein. License: BSD-3-Clause */
  (function () {
    'use strict'; var f, g = []; function l(a) { g.push(a); 1 == g.length && f() } function m() { for (; g.length;)g[0](), g.shift() } f = function () { setTimeout(m) }; function n(a) { this.a = p; this.b = void 0; this.f = []; var b = this; try { a(function (a) { q(b, a) }, function (a) { r(b, a) }) } catch (c) { r(b, c) } } var p = 2; function t(a) { return new n(function (b, c) { c(a) }) } function u(a) { return new n(function (b) { b(a) }) } function q(a, b) { if (a.a == p) { if (b == a) throw new TypeError; var c = !1; try { var d = b && b.then; if (null != b && "object" == typeof b && "function" == typeof d) { d.call(b, function (b) { c || q(a, b); c = !0 }, function (b) { c || r(a, b); c = !0 }); return } } catch (e) { c || r(a, e); return } a.a = 0; a.b = b; v(a) } }
    function r(a, b) { if (a.a == p) { if (b == a) throw new TypeError; a.a = 1; a.b = b; v(a) } } function v(a) { l(function () { if (a.a != p) for (; a.f.length;) { var b = a.f.shift(), c = b[0], d = b[1], e = b[2], b = b[3]; try { 0 == a.a ? "function" == typeof c ? e(c.call(void 0, a.b)) : e(a.b) : 1 == a.a && ("function" == typeof d ? e(d.call(void 0, a.b)) : b(a.b)) } catch (h) { b(h) } } }) } n.prototype.g = function (a) { return this.c(void 0, a) }; n.prototype.c = function (a, b) { var c = this; return new n(function (d, e) { c.f.push([a, b, d, e]); v(c) }) };
    function w(a) { return new n(function (b, c) { function d(c) { return function (d) { h[c] = d; e += 1; e == a.length && b(h) } } var e = 0, h = []; 0 == a.length && b(h); for (var k = 0; k < a.length; k += 1)u(a[k]).c(d(k), c) }) } function x(a) { return new n(function (b, c) { for (var d = 0; d < a.length; d += 1)u(a[d]).c(b, c) }) }; window.Promise || (window.Promise = n, window.Promise.resolve = u, window.Promise.reject = t, window.Promise.race = x, window.Promise.all = w, window.Promise.prototype.then = n.prototype.c, window.Promise.prototype["catch"] = n.prototype.g);
  }());

  (function () {
    function l(a, b) { document.addEventListener ? a.addEventListener("scroll", b, !1) : a.attachEvent("scroll", b) } function m(a) { document.body ? a() : document.addEventListener ? document.addEventListener("DOMContentLoaded", function c() { document.removeEventListener("DOMContentLoaded", c); a() }) : document.attachEvent("onreadystatechange", function k() { if ("interactive" == document.readyState || "complete" == document.readyState) document.detachEvent("onreadystatechange", k), a() }) }; function r(a) {
    this.a = document.createElement("div"); this.a.setAttribute("aria-hidden", "true"); this.a.appendChild(document.createTextNode(a)); this.b = document.createElement("span"); this.c = document.createElement("span"); this.h = document.createElement("span"); this.f = document.createElement("span"); this.g = -1; this.b.style.cssText = "max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;"; this.c.style.cssText = "max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;";
      this.f.style.cssText = "max-width:none;display:inline-block;position:absolute;height:100%;width:100%;overflow:scroll;font-size:16px;"; this.h.style.cssText = "display:inline-block;width:200%;height:200%;font-size:16px;max-width:none;"; this.b.appendChild(this.h); this.c.appendChild(this.f); this.a.appendChild(this.b); this.a.appendChild(this.c)
    }
    function t(a, b) { a.a.style.cssText = "max-width:none;min-width:20px;min-height:20px;display:inline-block;overflow:hidden;position:absolute;width:auto;margin:0;padding:0;top:-999px;white-space:nowrap;font-synthesis:none;font:" + b + ";" } function y(a) { var b = a.a.offsetWidth, c = b + 100; a.f.style.width = c + "px"; a.c.scrollLeft = c; a.b.scrollLeft = a.b.scrollWidth + 100; return a.g !== b ? (a.g = b, !0) : !1 } function z(a, b) { function c() { var a = k; y(a) && a.a.parentNode && b(a.g) } var k = a; l(a.b, c); l(a.c, c); y(a) }; function A(a, b) { var c = b || {}; this.family = a; this.style = c.style || "normal"; this.weight = c.weight || "normal"; this.stretch = c.stretch || "normal" } var B = null, C = null, E = null, F = null; function G() { if (null === C) if (J() && /Apple/.test(window.navigator.vendor)) { var a = /AppleWebKit\/([0-9]+)(?:\.([0-9]+))(?:\.([0-9]+))/.exec(window.navigator.userAgent); C = !!a && 603 > parseInt(a[1], 10) } else C = !1; return C } function J() { null === F && (F = !!document.fonts); return F }
    function K() { if (null === E) { var a = document.createElement("div"); try { a.style.font = "condensed 100px sans-serif" } catch (b) { } E = "" !== a.style.font } return E } function L(a, b) { return [a.style, a.weight, K() ? a.stretch : "", "100px", b].join(" ") }
    A.prototype.load = function (a, b) {
      var c = this, k = a || "BESbswy", q = 0, D = b || 3E3, H = (new Date).getTime(); return new Promise(function (a, b) {
        if (J() && !G()) { var M = new Promise(function (a, b) { function e() { (new Date).getTime() - H >= D ? b() : document.fonts.load(L(c, '"' + c.family + '"'), k).then(function (c) { 1 <= c.length ? a() : setTimeout(e, 25) }, function () { b() }) } e() }), N = new Promise(function (a, c) { q = setTimeout(c, D) }); Promise.race([N, M]).then(function () { clearTimeout(q); a(c) }, function () { b(c) }) } else m(function () {
          function u() {
            var b; if (b = -1 !=
              f && -1 != g || -1 != f && -1 != h || -1 != g && -1 != h) (b = f != g && f != h && g != h) || (null === B && (b = /AppleWebKit\/([0-9]+)(?:\.([0-9]+))/.exec(window.navigator.userAgent), B = !!b && (536 > parseInt(b[1], 10) || 536 === parseInt(b[1], 10) && 11 >= parseInt(b[2], 10))), b = B && (f == v && g == v && h == v || f == w && g == w && h == w || f == x && g == x && h == x)), b = !b; b && (d.parentNode && d.parentNode.removeChild(d), clearTimeout(q), a(c))
          } function I() {
            if ((new Date).getTime() - H >= D) d.parentNode && d.parentNode.removeChild(d), b(c); else {
              var a = document.hidden; if (!0 === a || void 0 === a) f = e.a.offsetWidth,
                g = n.a.offsetWidth, h = p.a.offsetWidth, u(); q = setTimeout(I, 50)
            }
          } var e = new r(k), n = new r(k), p = new r(k), f = -1, g = -1, h = -1, v = -1, w = -1, x = -1, d = document.createElement("div"); d.dir = "ltr"; t(e, L(c, "sans-serif")); t(n, L(c, "serif")); t(p, L(c, "monospace")); d.appendChild(e.a); d.appendChild(n.a); d.appendChild(p.a); document.body.appendChild(d); v = e.a.offsetWidth; w = n.a.offsetWidth; x = p.a.offsetWidth; I(); z(e, function (a) { f = a; u() }); t(e, L(c, '"' + c.family + '",sans-serif')); z(n, function (a) { g = a; u() }); t(n, L(c, '"' + c.family + '",serif'));
          z(p, function (a) { h = a; u() }); t(p, L(c, '"' + c.family + '",monospace'))
        })
      })
    }; "object" === typeof module ? module.exports = A : (window.FontFaceObserver = A, window.FontFaceObserver.prototype.load = A.prototype.load);
  }());


  var fontA = new FontFaceObserver('OpenSans');
  var fontB = new FontFaceObserver('OpenSansBold', {
    weight: 700
  });

  Promise.all([
    fontA.load(null, 10000),
    fontB.load(null, 10000)
  ]).then(function () {
    document.documentElement.className += " fonts-loaded";

    // Optimization for Repeat Views
    sessionStorage.fontsLoadedFoutWithClassPolyfill = true;
  });
})();
